guides/source/action_cable_overview.md

```diff
@@ -357,6 +365,64 @@ consumer.subscriptions.create({ channel: "ChatChannel", room: "1st Room" })
 consumer.subscriptions.create({ channel: "ChatChannel", room: "2nd Room" })
 ```
 
+Rails provides a generator to scaffold the server and client-side channel files:
+
+```bash
+$ bin/rails generate channel
+Usage:
+  rails generate channel NAME [method method] [options]
+
+...
+
+Example:
+========
+    bin/rails generate channel Chat speak
+
+    creates a Chat channel class, test and JavaScript asset:
+        Channel:    app/channels/chat_channel.rb
+        Test:       test/channels/chat_channel_test.rb
+        Assets:     $JAVASCRIPT_PATH/channels/chat_channel.js
+```
+
+#### Importing Channels
```

Comment:
I should also add that I'm open to feedback on this section. This was how I imported my channels and I assume how most would do it in their app.

We could add another entry for non-importmap setups, if we think it's valuable information.